qml/Silica4v-play/TextSwitch.qml

```diff
@@ -10,4 +10,6 @@ AppCheckBox {
     spacing: BVApp.Theme.paddingLarge
 
     labelColorOff: BVApp.Theme.primaryColor
+    // labelFontSize: BVApp.Theme.fontSizeExtraSmall
```

Comment:
Wanna leave this as comment?

Reply:
Agreed. Normally I am **really** strongly against commented out code. In this exceptional case, I'd like to keep it in as a reminder.